From 56aae12deaf3ac9b29fab9177601bb5af5d3fac5 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sat, 11 Nov 2023 17:00:34 +0000
Subject: [PATCH] Add CCache Check to Environment Setup

---
 build/envsetup.sh | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index faef80c1..e0011377 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -959,3 +959,20 @@ function fixup_common_out_dir() {
         mkdir -p ${common_out_dir}
     fi
 }
+
+# Check and set ccache path on envsetup
+if [ -z ${CCACHE_EXEC} ]; then
+	ccache_path=$(which ccache)
+	if [ ! -z "$ccache_path" ]; then
+		mkdir /mnt/ccache
+		mount --bind ~/.ccache /mnt/ccache
+		export USE_CCACHE=1
+		export CCACHE_COMPRESS=1
+		export CCACHE_EXEC=/usr/bin/ccache
+		export CCACHE_DIR=/mnt/ccache
+		ccache -M 50G -F 0
+		echo "ccache found and CCACHE_EXEC has been set to : $ccache_path"
+	else
+		echo "ccache not found/installed!"
+	fi
+fi
-- 
2.34.1

