From ddfda3316ab11257f7b05bd1d86336cc74c794f4 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sat, 1 Oct 2022 18:01:41 +0000
Subject: [PATCH 1/2] Remove Duplicated Flipendo

---
 prebuilt/google/apk/Android.mk | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/prebuilt/google/apk/Android.mk b/prebuilt/google/apk/Android.mk
index 79e247ec..b46aa4fc 100644
--- a/prebuilt/google/apk/Android.mk
+++ b/prebuilt/google/apk/Android.mk
@@ -14,12 +14,3 @@
 
 LOCAL_PATH:= $(call my-dir)
 
-include $(CLEAR_VARS)
-LOCAL_MODULE := Flipendo
-LOCAL_MODULE_TAGS := optional
-LOCAL_SRC_FILES := Flipendo.apk
-LOCAL_CERTIFICATE := platform
-LOCAL_MODULE_CLASS := APPS
-LOCAL_SYSTEM_EXT_MODULE := true
-LOCAL_MODULE_SUFFIX := $(COMMON_ANDROID_PACKAGE_SUFFIX)
-include $(BUILD_PREBUILT)
-- 
2.25.1

