From 05c7f473a3f45c10dd86b54a8e5a2e33c94a366b Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 3 Sep 2023 10:52:33 +0000
Subject: [PATCH] BQR

---
 config/superior_props.mk | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/config/superior_props.mk b/config/superior_props.mk
index aa8be3e3..4d04d5e0 100644
--- a/config/superior_props.mk
+++ b/config/superior_props.mk
@@ -43,4 +43,13 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 
 # SystemUI
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-    dalvik.vm.systemuicompilerfilter=speed
\ No newline at end of file
+    dalvik.vm.systemuicompilerfilter=speed
+
+# Override BQR mask to enable LE Audio Choppy report, remove BTRT logging
+ifneq (,$(filter userdebug eng, $(TARGET_BUILD_VARIANT)))
+PRODUCT_PRODUCT_PROPERTIES += \
+    persist.bluetooth.bqr.event_mask=262238
+else
+PRODUCT_PRODUCT_PROPERTIES += \
+    persist.bluetooth.bqr.event_mask=94
+endif
-- 
2.34.1

