From ae1a2d2fe2e1dca65aa5221bdb9fe612050d0127 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Wed, 28 Jun 2023 21:10:15 +0000
Subject: [PATCH] Enable R8 Optimizations

---
 config/phh_gsi.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/phh_gsi.mk b/config/phh_gsi.mk
index dccc6529..f864fa44 100644
--- a/config/phh_gsi.mk
+++ b/config/phh_gsi.mk
@@ -15,6 +15,10 @@ USE_CCACHE := true
 # Build APN
 PRODUCT_PACKAGES += apns-conf.xml
 
+# R8 Java optimizations for SystemUI and system_server
+SYSTEM_OPTIMIZE_JAVA ?= true
+SYSTEMUI_OPTIMIZE_JAVA ?= true
+
 # Set Build Variant prop for OTA
 ifeq ($(WITH_FULL_THEGAPPS), true)
     PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.phh.buildvariant=Secure
-- 
2.34.1

