From b79e42e7ed13dab68db2acd93ffa3c7637d09935 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 9 Oct 2022 14:26:45 +0000
Subject: [PATCH] Add Shared Blocks flag

---
 config/phh_gsi.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/phh_gsi.mk b/config/phh_gsi.mk
index 1fe40c1c..a2207ef5 100644
--- a/config/phh_gsi.mk
+++ b/config/phh_gsi.mk
@@ -13,6 +13,9 @@ TARGET_NO_KERNEL_OVERRIDE := true
 # Set Bootanimationn at 720P
 TARGET_BOOT_ANIMATION_RES := 720
 
+# Sharee Block (RO)
+BOARD_EXT4_SHARE_DUP_BLOCKS := false
+
 # Use CCache
 USE_CCACHE := true
 
-- 
2.34.1

