From b8dee96f48806376e98960d34891bf0b56914cf5 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Thu, 30 May 2024 19:28:02 -0300
Subject: [PATCH 02/31] Set a prop to define samsung proximity sensor

---
 .../server/display/DisplayPowerProximityStateController.java  | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/services/core/java/com/android/server/display/DisplayPowerProximityStateController.java b/services/core/java/com/android/server/display/DisplayPowerProximityStateController.java
index ee8ffe2f..76534cc3 100644
--- a/services/core/java/com/android/server/display/DisplayPowerProximityStateController.java
+++ b/services/core/java/com/android/server/display/DisplayPowerProximityStateController.java
@@ -25,6 +25,7 @@ import android.os.Handler;
 import android.os.Looper;
 import android.os.Message;
 import android.os.SystemClock;
+import android.os.SystemProperties;
 import android.util.Slog;
 import android.util.TimeUtils;
 import android.view.Display;
@@ -379,6 +380,7 @@ public final class DisplayPowerProximityStateController {
             for(Sensor sensor: sensors) {
                 if("com.samsung.sensor.physical_proximity".equals(sensor.getStringType()))
                     mProximitySensor = sensor;
+		    SystemProperties.set("persist.sys.phh.proximity", "samsung-physical");
             }
         }
         if(mProximitySensor == null) {
@@ -386,6 +388,7 @@ public final class DisplayPowerProximityStateController {
             for(Sensor sensor: sensors) {
                 if("com.samsung.sensor.hover_proximity".equals(sensor.getStringType()))
                     mProximitySensor = sensor;
+		    SystemProperties.set("persist.sys.phh.proximity", "samsung-hover");
             }
         }
         if(mProximitySensor == null) {
@@ -393,6 +396,7 @@ public final class DisplayPowerProximityStateController {
             for(Sensor sensor: sensors) {
                 if("com.samsung.sensor.touch_proximity".equals(sensor.getStringType()))
                     mProximitySensor = sensor;
+		    SystemProperties.set("persist.sys.phh.proximity", "samsung-touch");
             }
         }
         if (mProximitySensor != null) {
-- 
2.47.0

