From 7c3cf4362091c938120ea2eeba4f4715e27c07ed Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Wed, 11 Mar 2020 14:02:35 +0100
Subject: [PATCH] Every voldmanaged storage is adoptable

---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 6bad2d8..e50872c 100644
--- a/main.cpp
+++ b/main.cpp
@@ -274,7 +274,7 @@ static int process_config(VolumeManager* vm, VoldConfigs* configs) {
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 configs->has_adoptable = true;
             }
-- 
2.34.1

