From 4b6fd5084f7105ee6d540a74f61cea4f8a9ebda5 Mon Sep 17 00:00:00 2001
From: corneranchu <corneranchu@gmail.com>
Date: Sat, 1 Oct 2022 02:45:02 +0000
Subject: [PATCH] By-pass Vendor Calls

---
 target/product/aosp_arm64.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/target/product/aosp_arm64.mk b/target/product/aosp_arm64.mk
index 01897b77d..9be42756b 100644
--- a/target/product/aosp_arm64.mk
+++ b/target/product/aosp_arm64.mk
@@ -51,9 +51,9 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_product.mk)
 #
 # All components inherited here go to vendor or vendor_boot image
 #
-$(call inherit-product, $(SRC_TARGET_DIR)/product/emulator_vendor.mk)
-$(call inherit-product, $(SRC_TARGET_DIR)/board/generic_arm64/device.mk)
-$(call inherit-product, $(SRC_TARGET_DIR)/product/non_ab_device.mk)
+# $(call inherit-product, $(SRC_TARGET_DIR)/product/emulator_vendor.mk)
+# $(call inherit-product, $(SRC_TARGET_DIR)/board/generic_arm64/device.mk)
+# $(call inherit-product, $(SRC_TARGET_DIR)/product/non_ab_device.mk)
 
 #
 # Special settings for GSI releasing
-- 
2.34.1

