From c5d700fefa0c6960ea77c223b267ba6ff84309b2 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Mon, 17 Jun 2024 11:33:46 -0300
Subject: [PATCH] SurfaceFlinger: Disable SF HWC backpressure

---
 services/surfaceflinger/SurfaceFlinger.cpp | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/services/surfaceflinger/SurfaceFlinger.cpp b/services/surfaceflinger/SurfaceFlinger.cpp
index edc063618d..c0748983e1 100644
--- a/services/surfaceflinger/SurfaceFlinger.cpp
+++ b/services/surfaceflinger/SurfaceFlinger.cpp
@@ -2099,10 +2099,12 @@ bool SurfaceFlinger::commit(nsecs_t frameTime, int64_t vsyncId, nsecs_t expected
         }
     }
 
-    if (framePending && mPropagateBackpressure) {
-        if ((hwcFrameMissed && !gpuFrameMissed) || mPropagateBackpressureClientComposition) {
-            scheduleCommit(FrameHint::kNone);
-            return false;
+    if (framePending && base::GetBoolProperty("persist.sys.phh.enable_sf_hwc_backpressure"s, true)) {
+        if (framePending && mPropagateBackpressure) {
+            if ((hwcFrameMissed && !gpuFrameMissed) || mPropagateBackpressureClientComposition) {
+                scheduleCommit(FrameHint::kNone);
+                return false;
+	    }
         }
     }
 
-- 
2.34.1

