From 7667ab219d62c3d2355a6298fbfb2d0f5c27dbd4 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 7 Oct 2022 04:16:13 +0000
Subject: [PATCH] generate.sh: Replace me.phh.superuser with SuperUser

To fit with Dynamic Root Patches
---
 generate.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generate.sh b/generate.sh
index fac8208..7afe4e2 100644
--- a/generate.sh
+++ b/generate.sh
@@ -54,7 +54,7 @@ for part in a ab;do
 				su_suffix='N'
 				if [ "$su" == "yes" ];then
 					su_suffix='S'
-					extra_packages+=' phh-su me.phh.superuser'
+					extra_packages+=' phh-su SuperUser'
 				fi
 
 				part_suffix='a'
-- 
2.34.1

