From 580f2f521b0ea5b1688e1853c5ce873a330a6dc9 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Fri, 2 Feb 2024 16:51:36 -0300
Subject: [PATCH] Simplify Securize

---
 phh-securize.sh | 30 ++++++++----------------------
 1 file changed, 8 insertions(+), 22 deletions(-)

diff --git a/phh-securize.sh b/phh-securize.sh
index 864c3d2..9234459 100644
--- a/phh-securize.sh
+++ b/phh-securize.sh
@@ -1,39 +1,25 @@
 #!/system/bin/sh
 
-if [ -e /system/bin/magisk ]
-then
-    # remove bind-mount of phh-su overriding /system/bin/su -> ./magisk
-    umount -l /system/bin/magisk
-    # we need to modify the real system partition
-    MAGISK_MIRROR="$(magisk --path)/.magisk/mirror"
-    SYSTEM=$MAGISK_MIRROR/system
-    MOUNTPOINT_LIST="$MAGISK_MIRROR/system_root $MAGISK_MIRROR/system"
-else
-    SYSTEM=/system
-    MOUNTPOINT_LIST="/system /"
-fi
-
-# remove bind-mount of phh-su (preventing $SYSTEM/xbin/su to be removed)
+# Remove bind-mount of phh-su
+umount -l /system/bin/magisk
 umount -l /system/xbin/su
+SYSTEM=/system
+MOUNTPOINT_LIST="/system /"
 
 for MOUNTPOINT in $MOUNTPOINT_LIST
 do
     [ -d $MOUNTPOINT ] && mountpoint -q $MOUNTPOINT && break
 done
 
-mount -o remount,rw $MOUNTPOINT
-remount
+mount -o rw,remount $MOUNTPOINT
 
-touch $SYSTEM/phh/secure
 rm $SYSTEM/xbin/su
 rm $SYSTEM/bin/phh-su
 rm $SYSTEM/etc/init/su.rc
 rm $SYSTEM/bin/phh-root.sh
-rm -Rf $SYSTEM/priv-app/SuperUser
+rm -rf $SYSTEM/priv-app/SuperUser
 rm $SYSTEM/bin/phh-securize.sh
-rm -Rf /data/su || true
-mount -o remount,ro $MOUNTPOINT
+rm -rf /data/su || true
+mount -o rw, remount $MOUNTPOINT
 sync
-mkdir /metadata/phh
-touch /metadata/phh/secure
 reboot
-- 
2.34.1

