From e7ccc9aac62f08c354b37eabacccd563fc10bea6 Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 16 Oct 2022 13:58:32 +0000
Subject: [PATCH] Add lux props for moto e6 plus

---
 rw-system.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/rw-system.sh b/rw-system.sh
index 7b14f25..e73b5a0 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -1131,5 +1131,11 @@ if getprop ro.vendor.build.fingerprint |grep -iq redmi/camellia;then
 	setprop persist.sys.qcom-brightness 4095
 fi
 
+if getprop ro.product.vendor.device | grep -iq pokerp; then
+	# tinno auto brightness feature
+	resetprop_phh ro.smartabc_support true
+	resetprop_phh vendor.tinno.sensor_lux_map "0,10,30,80,160,300,520,700,1000,2000,4000"
+fi
+
 mount -o bind /mnt/phh/empty_dir /vendor/app/qti-logkit
 mount -o bind /mnt/phh/empty_dir /vendor/app/qti-logkit-lite
-- 
2.34.1

