From 9d7396591e6278aafb71ca541ad188223ea26a2f Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Sun, 29 Oct 2023 15:57:19 +0000
Subject: [PATCH] Nuke androidx, it breaks Settings UI

---
 base.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/base.mk b/base.mk
index c32f3d8..281df6c 100644
--- a/base.mk
+++ b/base.mk
@@ -265,7 +265,5 @@ PRODUCT_PACKAGES += \
 	MtkInCallService \
 
 # Two-pane layout in Settings
-PRODUCT_PACKAGES += \
-    androidx.window.extensions
 
 PRODUCT_EXTRA_VNDK_VERSIONS += 28
-- 
2.34.1

