From f9e03bb9ec89b48f38d23da74eb632e5c572b4ed Mon Sep 17 00:00:00 2001
From: ChonDoit <thphantomblog@gmail.com>
Date: Mon, 19 Jun 2023 22:31:42 +0000
Subject: [PATCH 2/5] Allow Huawei offline charging sepolicy

---
 sepolicy/charger.te | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sepolicy/charger.te b/sepolicy/charger.te
index cbcb255..53044d5 100644
--- a/sepolicy/charger.te
+++ b/sepolicy/charger.te
@@ -3,8 +3,8 @@ attribute sysfs_backlight_attr;
 allow charger sysfs_backlight_attr:file rw_file_perms;
 
 # Allow charger to write to sysfs_led_attr (only for huawei)
-# attribute sysfs_led_attr;
-# allow charger sysfs_led_attr:file rw_file_perms;
+attribute sysfs_led_attr;
+allow charger sysfs_led_attr:file rw_file_perms;
 
 # Allow charger to read and write to sysfs_power
 allow charger sysfs_power:file rw_file_perms;
-- 
2.34.1

